File: Leetcode/Cpp/solved/output/2696-Minimum String Lenght After Removing Substrings.cpp
class Solution {
public:
    int minLength(std::string s) {
        while (s.find("AB") != std::string::npos || s.find("CD") != std::string::npos) {
            size_t pos = s.find("AB");
            if (pos != std::string::npos) {
                s.erase(pos, 2);
            }
            pos = s.find("CD");
            if (pos != std::string::npos) {
                s.erase(pos, 2);
            }
        }
        return s.length();
    }
};

File: Leetcode/Java/solved/2696-Minimum String Lenght After Removing Substrings.java
public class Solution {
    public int minLength(String s) {
        while (s.contains("AB") || s.contains("CD")) {
            String temp = s.replace("AB", "");
            temp = temp.replace("CD", "");
            s = temp;
        }
        return s.length();
    }
}